docs/index.md

```diff
@@ -1,148 +1,125 @@
+    regression in production, revert the commit, and release the "new"
+    version
+
+## Concepts
+
+Atomist is all about making you more productive by helping you to
+automate away all the things that slow you down.  The following
+concepts describe the pieces Atomist provides to make it easy for you
+to create automations that reduce drudgery and distractions.
+
+### Development automation platform
+
+Atomist's development automation platform is powered by a service that
+ingests and correlates events from your software development flow.  At
+the heart of the service is a single coherent model of your
+development flow: code, people, and processes.  You query and mutate
+that model using [GraphQL][gql] when you write your own automations.
+
+[gql]: http://graphql.org/ (GraphQL)
+
+### Automation client
+
+Code in automations is written in Atomist automation clients, which
+interact with the Atomist platform using primarily GraphQL.  Each
+client is itself a server which hosts automations that can be invoked
+via the Atomist bot, Slack buttons, or when events occur.  A client
+can host any number of automations, and can be hosted wherever the
+author likes: locally during testing, inside a corporate firewall, or
+on a public cloud or PaaS.
+
+### Commands
+
+Commands perform actions when invoked by users by sending the Atomist
+bot messages, click buttons in Slack, or using the Atomist CLI.
+Command implementations are defined
+in [**command handlers**][command].
+
+Examples of commands include the bot command `create issue` and the
+`merge pull request` command that users invoke by pressing a button in
+Slack.
+
+[command]: automations/commands.md (Bot Commands)
+
+### Events
+
+Events can be anything from commits to pushes to CI builds to
+deployments to stack traces in production logs.  When an event is
+ingested, Atomist relates it to other events to build up necessary
+contextual information.  Atomist may also take automated action. The
+response to an event is defined in
+an [**event handler**][event].
+
+How does a handler know what event it should act on?  GraphQL
+subscriptions are used to match trigger criteria.  For example, you
+can create a subscription to receive an event when Kubernetes
+deployment results in a pod crash looping and connect that to an event
+handler that automatically rolls that deployment back and notifies the
+committers of the failure.
+
+[event]: automations/events.md (Event Automations)
+
+### Integrations
+
+Atomist receives events from and performs operations on many systems,
+including version control, CI systems, Slack, and more.
+
+Atomist uses the native integration technology for each platform or
+tool.  For example, to integrate with GitHub and Travis CI, Atomist
+uses webhooks; to integrate with Slack it uses their native real-time
+messaging (RTM) API.  For each platform Atomist integrates with, it
+asks for the minimal set of permissions required to perform its
```

Comment:
drop "its duties"

Reply:
Just "required to perform"? That seems abrupt.

Reply:
Maybe:

"For each platform Atomist integrates with it requests smallest set of permissions that are required to work"

Or drop the whole sentence? It's actually quite a high bar and easy to miss.

Also in addition to the 3 other comments above I notice that it's marked 5 more as outdated that aren't outdated - not sure why!

Reply:
You made them while I was fixing @kipz comments on/around the same lines. I noticed this too, expanded the comments, and addressed them all, I think. This one, I'm inclined to leave for @WillGardella to do something about.